### PR TITLE
add ngreact@0.3.0 w/ npm autoupdate

### LINKED ISSUE #8691: [Request] Add ng-re
**Library name:** ng-react
**Git repository url:** https://github.com/ngReact/ngReact.git
**npm package url(optional):** 
**License(s):** MIT
**Official homepage:** https://github.com/ngReact/ngReact
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #8691 
@kennynaoh Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ngReact/ngReact.git
 * Official website (optional, not the repository): https://ngReact.github.io/ngReact
 * NPM package url (optional):https://www.npmjs.com/package/ngreact
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 2035
   - Count of watchers: 86
   - Count of forks: 178
 * NPM download stats (optional):
   - Downloads in the last day: 23
   - Downloads in the last week: 2291
   - Downloads in the last month: 10429

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8691, cc @hasdavidc

### CODE REVIEW

File: ajax/libs/ngreact/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "ngreact",
+  "filename": "ngReact.min.js",
+  "author": "David Chang",
+  "version": "0.3.0",
+  "description": "Use React Components in Angular",
+  "homepage": "https://davidchang.github.io/ngReact/",
```
Comment (kennynaoh) [inline ajax/libs/ngreact/package.json]: broken website

**Comment by pvnr0082t** [comment]
@kennynaoh I have fixed homepage. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
@PeterDaveHello 
I think this PR is correct.
Please help me review it again.
Thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t I will tag another author who is active recently, the top 1 author seems to be not active recently:
https://github.com/ngReact/ngReact/graphs/contributors?from=2015-08-05&to=2016-08-09&type=c

**Pull request merged**